tempest/api/workloadmgr/functional/test_functional.py

```diff
@@ -53,7 +53,7 @@ def multiple_vms(self, vm_count, key_pairs, sec_groups):
                 vm_id = self.create_vm(security_group_id=random.choice(sec_groups), key_pair=kptouple[1], key_name=kptouple[0], vm_cleanup=True)
                 vms[vm_id] = [kptouple[0]]
             else:
-                boot_volume_id = self.create_volume(image_id=CONF.compute.image_ref, volume_cleanup=True)
+                boot_volume_id = self.create_volume(size=tvaultconf.bootfromvol_vol_size, image_id=CONF.compute.image_ref, volume_cleanup=True)
```

Comment:
Make the default size as CONF.volume.volume_size . Wherever required, you can pass tvaultconf.bootfromvol_vol_size

Reply:
yes this is according to that only i think... in base.py for create _volume method volume_size is taken as CONF.volume.volume_size only.... here i need bootfromvol_vol_size so used it

Reply:
Okay.